### ISSUE TITLE
[DataCap Application] <The Second Life> - <3D-printed new2>

### ISSUE THREAD

**Issue by AlterEgogo** [NDLABS-Leo/Allocator-Pathway-Enterprise-data#15]
### Version

1

### DataCap Applicant

The Second Life new2

### Project ID

007

### Data Owner Name

The Second Life new2

### Data Owner Country/Region

China

### Data Owner Industry

IT & Technology Services

### Website

https://mp.weixin.qq.com/s/IMtL_m4sQAxqCmNWlw28Gw

### Social Media Handle

https://mp.weixin.qq.com/s/IMtL_m4sQAxqCmNWlw28Gw

### Social Media Type

Slack

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

500TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

1PiB

### On-chain address for first allocation

f15tuvkdu3hp5g4y6ufrjttseppvqvfodinn2u23q

### Data Type of Application

Private Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
The Second Life Universe Block is a digital hipster block with 3D printing and digital twins as the underlying technology, and a collection of 3D-printed real-life artefacts, 3D-printed coffee, and meta-universe experiences. Embrace the future, private ordering, self-realisation, and one-click construction of the second life of real human beings.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
For each 3D printed model, we need to take high-precision photos of the 360-degree panorama. The production photos for one model are up to tens of thousands of photos, and we will model the model in 3D. We need to store these photos, models and other data
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We currently store our data on our own work computers, and on servers in the server room. In the case of data from the computers we will transfer it over the network, and in the case of data from the server we can do hard drive mailing
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
We have not previously stored on an IPFS network and the cost of storing on an IPFS network will make it much cheaper for us to do business.SP's information is something we are currently in contact with on an ongoing basis, and I will disclose it below in the request form.
```

### Please share a sample of the data

```text
You can check out the WeChat video number, the name of the account is "第二人生3D真人手办".
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [ ] I confirm

### If you chose not to confirm, what was the reason

```text
Because the data stored is corporate data, it has a certain level of confidentiality
```

### What is the expected retrieval frequency for this data

Never

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America

### How will you be distributing your data to storage providers

HTTP or FTP server, IPFS

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f03079511 HK；
f02327534 US；
f02894855 China；
f02826234 SG
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by NDLABS-Leo** [comment]
There is a system bug in issues that prevents signatures.

Please check the review log: https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/issues/10

**Comment by NDLABS-Leo** [comment]
Since the bugs have been taken care of, I signed it.
Please check the review log: https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/issues/10

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by AlterEgogo** [comment]
Nodal disclosure of co-operation：

f02827151 Guangdong
f02827694 Guangdong
f01926635 Sichuan
f02980903 KR
f01999119 Sichuan

**Comment by NDLABS-Leo** [comment]
![image](https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/assets/104019147/975656f4-6941-426b-9e4b-4065b514f6ea)
![image](https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/assets/104019147/dcbab852-464e-46fe-957d-ba4f846551af)
![image](https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/assets/104019147/43b1fcfa-96dd-466b-afa0-87c74d51a7f7)
![image](https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/assets/104019147/abd86fab-9748-4161-ac2b-5d4e15de4d6c)

**Comment by NDLABS-Leo** [comment]
Node connectivity testing complete

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by FilscanOfficial** [comment]
@AlterEgogo 
I have reviewed your application and you need to provide additional KYB and KYC documents. You can send an email to filscanteam@outlook.com with your company's KYB information and SP's KYC information. Thank you.

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
@AlterEgogo 
Currently, these two nodes appear to be unable to perform retrieval based on the report. Is there a solution for this？
<img width="1124" alt="截屏2024-10-03 22 15 04" src="https://github.com/user-attachments/assets/9ddfc6de-4cfd-47c9-9cc2-84a5b12d52ae">

**Comment by AlterEgogo** [comment]
@NDLABS-Leo 
7151and0903.
They have all used the Louts technical pathway and are currently unable to perform retrieval via Spark. We have paused our collaboration with them. Please review the following nodes, as we will select them for the next round of collaboration:
f02826815
f02828269
f02826253
f02825282

**Comment by NDLABS-Leo** [comment]
f02826815  | 152.32.253.7
<img width="334" alt="image" src="https://github.com/user-attachments/assets/27584b0c-7cc1-4a53-b4f1-4daaef8ee7ca">
f02828269  | 202.105.141.166
<img width="295" alt="image" src="https://github.com/user-attachments/assets/87094e75-d926-4866-87ee-6b278fdb4583">

f02826253 | 118.194.234.162
<img width="324" alt="image" src="https://github.com/user-attachments/assets/e8107f8f-1f6f-4104-b80a-5471e68ef633">

f02825282  | 39.109.85.3
<img width="293" alt="image" src="https://github.com/user-attachments/assets/451c7568-caa9-4577-9fc9-135a27869f7d">

**Comment by NDLABS-Leo** [comment]
<img width="504" alt="image" src="https://github.com/user-attachments/assets/65891036-a6a5-4eee-ad48-945696b30966">

@AlterEgogo 
The address review is fine, and the retrieval rate is normal. We hope to maintain this.

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
@AlterEgogo 
Hello, I noticed that most of the nodes have satisfactory retrieval rates; however, there are a few nodes with a retrieval rate of 0. Could you please explain the reasons behind this issue and let us know if there are any plans or solutions to improve it?
<img width="1096" alt="image" src="https://github.com/user-attachments/assets/5a1f7df8-8aa0-4aa5-8ddd-1a687d6cc3a5">
<img width="1120" alt="image" src="https://github.com/user-attachments/assets/72411e80-217d-43fb-8162-0522ffb49b55">

The nodes that need improvement are as follows:
<img width="1123" alt="image" src="https://github.com/user-attachments/assets/485d9cb4-d342-4d15-9d28-c2b839fde3ea">
<img width="1126" alt="image" src="https://github.com/user-attachments/assets/3f3cda68-6e9a-4c5f-9bc8-74453c724cd0">

**Comment by NDLABS-Leo** [comment]
2 storage providers sealed too much duplicate data - [f02828269](https://filfox.info/en/address/f02828269): 27.37%, [f02825282](https://filfox.info/en/address/f02825282): 27.17%

Note: This issue is a common problem and is due to a bug in the bot, which has existed in the community for a long time. It will not be considered in this review.

**Comment by AlterEgogo** [comment]
@NDLABS-Leo 
Yes, before selecting new SPs, we require them to meet a certain retrieval rate. Regarding the nodes with lower retrieval rates that you mentioned, some of them are experiencing network issues and are currently being adjusted. Additionally, node 0903 will not be used for data storage in the future. We hope you can allow us to continue our project, as this would be very helpful for us.

**Comment by AlterEgogo** [comment]
@NDLABS-Leo 
For the high-quality SPs we currently collaborate with, we plan to continue retaining them. At this time, we do not intend to introduce new SPs. However, should we establish partnerships with new SPs in the future, I will disclose this information in the Issues section as soon as possible. Thank you.

**Comment by NDLABS-Leo** [comment]
@AlterEgogo Understood, please continue using the DataCap responsibly.

**Comment by AlterEgogo** [comment]
@NDLABS-Leo Understood, thank you, sir.

**Comment by AlterEgogo** [comment]
New sp：f02826007 base Singapore

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger
